web_src/js/features/tableSort.js

```diff
@@ -0,0 +1,72 @@
+export default function initTableSort() {
+  $('th').each(function () {
+    // if data attribute sortt is set
+    if ($(this)[0].dataset.sortt) {
+      // get data
+      let data = $(this)[0].dataset.sortt;
+      data = data.split(',');
+
+      // add onclick event
+      $(this).on('click', () => {
+        tableSort(data[0], data[1], data[2]);
```

Comment:
If it's always those 3 args, name them outside the event:

```js
const [normsort, revsort, isdefault] = data.split(',').filter(value => !!value)

Reply:
they can be onyl 2 as third is optional (if set and true it means this colum is sorted by default)

Reply:
See destructuring example below. I think you can just take a `undefined` for the optional parameter as a `false`.